testapps/ApplicationUsingRelativePaths/Controllers/HomeController.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace ApplicationUsingRelativePaths.Controllers
+{
+    public class HomeController : Controller
+    {
+        public IActionResult Index()
+        {
+            return View("/Views/Home/Index.cshtml");
```

Comment:
This is an absolute path. Suggest using a relative path here as well, not just for the layout.